alerts/openshift-virtualization-operator/KubevirtHCOSingleStackIPv6.md

```diff
@@ -11,13 +11,25 @@ You cannot create virtual machines.
 
 ## Diagnosis
 
-- Obtain the network details for the cluster by running the following command:
+- Check the cluster network by running the following command:
   ```shell
-  $ oc describe network cluster
+  $ oc describe network.config/cluster
```

Comment:
```suggestion
  $ oc get network.config cluster -o yaml
```

The format of the output bellow is YAML, so `oc get` must be used to get it

Reply:
`oc get network cluster -o yaml` works for me too. But this way we at least won't get it mixed up with `operator.openshift.io/v1`